maintainers/maintainer-list.nix

```diff
@@ -13990,6 +13990,13 @@
     githubId = 2666479;
     name = "Y Nguyen";
   };
+  superherointj = {
+    name = "Sérgio G.";
+    email = "5861043+superherointj@users.noreply.github.com";
```

Comment:
Use of `users.noreply.github.com` is discouraged as in `lib/tests/maintainer.nix`

Reply:
Done